### PR TITLE
view lock

### PR DESCRIPTION
**Pull request by ghost**
I added a view lock bool.  Just needed a way to lock the camera and thought it might be helpful for others.  Works great if you're viewing a video or anything else that you need to constantly look at.

### CODE REVIEW

File: src/ofxOculusRift.h [hunk 1]
```diff
@@ -35,6 +35,7 @@ class ofxOculusRift
 	
 	bool setup();
 	bool isSetup();
+	bool lockView;
```
Comment (obviousjim) [inline src/ofxOculusRift.h:38]: is lock view initialized to false in the constructor?

**Comment by ghost** [comment]
oops, i forgot to add but yeah it should be.  should i add and submit a new pr?

**Comment by obviousjim** [comment]
if you push a new commit it will automaticall update the PR

On Thu, Oct 17, 2013 at 5:19 PM, Jason Walters notifications@github.comwrote:

> oops, i forgot to add but yeah it should be. should i add and submit a new
> pr?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/obviousjim/ofxOculusRift/pull/1#issuecomment-26553966
> .

## 
- James

**Comment by ghost** [comment]
done!

**Comment by obviousjim** [comment]
awesome!

**Pull request merged**